File: lista3/zadanie1/Romb.java
class Romb extends Czworokat  {

    public Romb(int bok1, int bok2, int bok3, int bok4, int kat){
        super(bok1, bok2, bok3, bok4, kat);
        
    }

    public double obliczPole() {
        return  (dajBok1() * dajBok2() * Math.sin(dajKat()) );

    }

    public void dajNazwe(){
        System.out.println("Jestem rombem");
    }
    
}

File: lista3/zadanie2/Romb.cpp
#include "Figura.hpp"
#include "Czworokat.hpp"
#include "Romb.hpp"


double Romb::obliczPole(){
    return dajBok1() * dajBok1()* sin(dajKat());
}

void Romb::dajNazwe(){
    std::cout << "Jestem rombem" << std::endl;
}


